src/Files/Files.csproj

```diff
@@ -1045,7 +1045,7 @@
     <Content Include="Assets\WSL\ubuntu.svg" />
     <Content Include="Assets\WSL\ubuntupng.png" />
     <Content Include="Properties\Default.rd.xml" />
-    <None Include="MultilingualResources\Files.nb-NO.xlf" />
+    <XliffResource Include="MultilingualResources\Files.nb-NO.xlf" />
```

Comment:
?

Reply:
Needed so Multilingual app toolkit recognizes it.